sekoia_automation/trigger.py

```diff
@@ -89,7 +89,6 @@ def stop(self, *args, **kwargs):  # noqa: ARG002
         """
         # Exit signal received, asking the processor to stop
         self._stop_event.set()
-        self.stop_monitoring()
```

Comment:
Is this change intended ?

Reply:
The revert of the previous commit caused it. I'm not sure how useful it is to have it.